.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Scope 
+ - Ready to review
+ - In Progress
+
+Description....
+
+### Area of Application / Project Impacted 
+
+
+
+### Related issues & PR's
+
+ - closes #
+ - related to #
+
+## Steps to Test
+
+ - ..
+ - .
+
+## TODO : 
+ - [x] Create PR
+ - [ ] First Review
+ - [ ] Second Review
+ - [ ] task to complete
+ - [ ] task...
```

Comment:
I feel that this is perhaps too much information required for a pull request. A lot of this stuff is being duplicated from the issue into the PR and we will just be creating too much admin to do per pull request. 

I feel a PR could be much simpler while still maintaining enough information. 

## Problem

## Solution

## Test

cc: <tag people you want notified of the PR>